mm2src/mm2_bitcoin/serialization/src/reader.rs

```diff
@@ -57,29 +57,40 @@ pub enum CoinVariant {
     LBC,
     Standard,
     PPC,
+    /// Needed to deserialize `RICK` block headers correctly as it's genesis header is `version 1` while all other
+    /// block headers are `version 4`.
+    RICK,
+    /// Same reason as RICK.
+    MORTY,
 }
 
 impl CoinVariant {
     pub fn is_btc(&self) -> bool { matches!(self, CoinVariant::BTC) }
-
+    pub fn is_btc_or_ppc(&self) -> bool { matches!(self, CoinVariant::BTC | CoinVariant::PPC) }
```

Comment:
I think better not to unite the checks and leave the as `is_btc` and `is_ppc`

Reply:
done